Sources/DurationPickerSwiftUI/Public/DurationPickerView.swift

```diff
@@ -0,0 +1,188 @@
+  @_documentation(visibility: internal)
+  public typealias UIViewType = DurationPicker
+  
+  /// The components displayed by the duration picker.
+  ///
+  /// The mode determines which combination of hours, minutes, and seconds are displayed. You can set and retrieve the mode value through the ``DurationPickerView/mode`` property.
+  public typealias Components = DurationPicker.Mode
+    
+  @Binding private var duration: TimeInterval
+  
+  private var mode: Components
+  
+  private var hourInterval: Int
+  private var minuteInterval: Int
+  private var secondInterval: Int
+  private var minumumDuration: TimeInterval?
+  private var maximumDuration: TimeInterval?
+ 
+  // This has to be public to comply with UIViewRepresentable, but we don't actually want it to show up in the doc.
+  @_documentation(visibility: internal)
+  public func makeUIView(context: Context) -> DurationPicker {
+    let timeDurationPicker = DurationPicker()
+    
+    timeDurationPicker.pickerMode = mode
+    timeDurationPicker.hourInterval = hourInterval
+    timeDurationPicker.minuteInterval = minuteInterval
+    timeDurationPicker.secondInterval = secondInterval
+    timeDurationPicker.minimumDuration = minumumDuration
+    timeDurationPicker.maximumDuration = maximumDuration
+    
+    timeDurationPicker.addTarget(context.coordinator, action: #selector(Coordinator.changed(_:)), for: .primaryActionTriggered)
+    return timeDurationPicker
+  }
+
+  // This has to be public to comply with UIViewRepresentable, but we don't actually want it to show up in the doc.
+  @_documentation(visibility: internal)
+  public func updateUIView(_ uiView: DurationPicker, context: Context) {
+    if (uiView.duration != duration) {
+      uiView.duration = duration
+    }
+    if (uiView.pickerMode != mode) {
+      uiView.pickerMode = mode
+    }
+    if (uiView.hourInterval != hourInterval) {
+      uiView.hourInterval = hourInterval
+    }
+    if (uiView.minuteInterval != minuteInterval) {
+      uiView.minuteInterval = minuteInterval
+    }
+    if (uiView.secondInterval != secondInterval) {
+      uiView.secondInterval = secondInterval
+    }
+    if (uiView.minimumDuration != minumumDuration) {
+      uiView.minimumDuration = minumumDuration
+    }
+    if (uiView.maximumDuration != maximumDuration) {
+      uiView.maximumDuration = maximumDuration
+    }
+  }
+
+  // This has to be public to comply with UIViewRepresentable, but we don't actually want it to show up in the doc.
+  @_documentation(visibility: internal)
+  public func makeCoordinator() -> DurationPickerView.Coordinator {
+    return Coordinator(duration: $duration)
+  }
+  
+  // This has to be public to comply with UIViewRepresentable, but we don't actually want it to show up in the doc.
+  @_documentation(visibility: internal)
+  public class Coordinator: NSObject {
+    private var duration: Binding<TimeInterval>
```

Comment:
Nit add vertical space
```suggestion

    private var duration: Binding<TimeInterval>
```